theforeman.org/pipelines/release/source/katello.groovy

```diff
@@ -69,10 +69,9 @@ pipeline {
                         expression { fileExists('package.json') }
                     }
                     steps {
-                        sh(script: "npm install --package-lock-only --no-audit --legacy-peer-deps", label: "npm install --package-lock-only")
-                        archiveArtifacts(artifacts: 'package-lock.json')
```

Comment:
We should archive `package-lock.json` after line 48. Mind doing that while you're at it?